Frontend/ui-driver/src/Services/Backend.purs

```diff
@@ -393,14 +393,15 @@ getDriverInfoBTGet = do
 getDriverInfoBTPost :: Boolean -> FlowBT String GetDriverInfoResp
 getDriverInfoBTPost enabled = do
     headers <- getHeaders' "" true
-    (UpdateFeatureInDInfoResp resp1) <- withAPIResultBT ((EP.getDriverInfoV2 "" )) identity errorHandler (lift $ lift $ callAPI headers  (DriverInfoReq {isAdvancedBookingEnabled : Just enabled}))
+    let config = getAppConfig appConfig
+    (UpdateFeatureInDInfoResp resp1) <- withAPIResultBT ((EP.getDriverInfoV2 "" )) identity errorHandler (lift $ lift $ callAPI headers  (DriverInfoReq {isAdvancedBookingEnabled : Just enabled, isInteroperable: Just config.feature.enableInterOperability}))
     pure (resp1)
     where
         errorHandler (ErrorPayload errorPayload) =  do
                 BackT $ pure GoBack
 
-getDriverInfoApi :: DriverInfoReq -> Flow GlobalState (Either ErrorResponse GetDriverInfoResp)
-getDriverInfoApi payload = do
+getDriverInfoApi :: String -> Flow GlobalState (Either ErrorResponse GetDriverInfoResp)
+getDriverInfoApi dummy = do
```

Comment:
we can use underscore if dummy is not being used.

Reply:
underscore will act similar to when no args passed, so `dummy` is required